runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -403,4 +403,27 @@ default boolean textOutline()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+			keyName = "highlightStackable",
+			name = "Highlight stackable in inventory",
+			description = "Highlight stackable ground items already in inventory",
+			position = 30
+	)
+	default boolean highlightStackable()
```

Comment:
Typically for things like this we'd just make the color default to null, eg:

```java

	@Alpha
	@ConfigItem(
		keyName = "highlightStackableColor",
		name = "Highlight stackable",
		description = "Configures the color for highlighted stackable items",
		position = 31
	)
	Color highlightStackableColor();
```

and then null-test it to see if it is enabled or not, instead of having a separate config option.